lib/MPIABI/src/MPIABI.jl

```diff
@@ -0,0 +1,79 @@
+module MPIABI
+
+using Preferences
+
+function known_abis()
+    return (:MicrosoftMPI, :MPICH, :OpenMPI, :MPItrampoline)
```

Comment:
should we make these all lowercase?

Reply:
I think JLLWrapper&Co normalize these to lower-case. I don't have a strong preference.